<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QTabWidget" name="triggerTypeTabs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>881</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="manual_tab">
    <attribute name="title">
     <string>Manual interval trigger</string>
    </attribute>
    <widget class="QGroupBox" name="manualAttributeGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>801</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>Feature Intervals</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="nn_tab">
    <attribute name="title">
     <string>NN trigger</string>
    </attribute>
    <widget class="QGroupBox" name="nnHyperparamGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>791</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Import hyperparameter file:</string>
     </property>
     <widget class="QLineEdit" name="nnHyperparamLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="nnBrowseHyperparamButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="nnClassesGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>611</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Select Classes For Trigger</string>
     </property>
     <widget class="QListWidget" name="nnClassListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="nnStartTrainingButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Training</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nnSaveModelButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Trained Model</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="lvq_tab">
    <attribute name="title">
     <string>LVQ trigger</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="browseInputButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputFileLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>&lt;existing &quot;.wf&quot; file&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select input file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadInputButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load selected file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
